src/root/components/FinalReportExport/FinalReportPdfDocument/index.tsx

```diff
@@ -0,0 +1,828 @@
+          <Text>
+            {strings.finalReportIndicatorTargetLabel}
+          </Text>
+        </View>
+        <View style={pdfStyles.piHeaderCell}>
+          <Text>
+            {strings.finalReportIndicatorActualLabel}
+          </Text>
+        </View>
+      </View>
+
+      {
+        data?.indicators?.map((indicator) => (
+          <View
+            style={pdfStyles.piRow}
+            key={indicator.id}
+          >
+            <View style={pdfStyles.piContentCell}>
+              <Text style={pdfStyles.piBorderCell}>
+                {indicator?.title}
+              </Text>
+            </View>
+            <View style={[pdfStyles.piHeaderCell, pdfStyles.fontWeightNormalAndSmall]}>
+              <Text style={pdfStyles.fontWeightNormalAndSmall}>
+                {indicator?.target}
+              </Text>
+            </View>
+            <View style={[pdfStyles.piHeaderCell, pdfStyles.fontWeightNormalAndSmall]}>
+              <Text style={pdfStyles.fontWeightNormalAndSmall}>
+                {indicator?.actual}
+              </Text>
+            </View>
+          </View>
+        ))
+      }
+      <View style={pdfStyles.piRow}>
+        <Text style={[pdfStyles.piBorderCell, pdfStyles.fontWeightBoldAndLarge]}>
+          {strings.finalReportPlannedInterventionNarrativeAchievement}
+        </Text>
+      </View>
+      {data?.challenges && (
+        <View style={pdfStyles.piRow}>
+          <Text style={pdfStyles.piBorderCell}>
+            {data?.narrative_description_of_achievements}
+          </Text>
+        </View>
+      )}
+      <View style={pdfStyles.piRow}>
+        <Text style={[pdfStyles.piBorderCell, pdfStyles.fontWeightBoldAndLarge]}>
+          {strings.finalReportPlannedInterventionChallenges}
+        </Text>
+      </View>
+      {data?.challenges && (
+        <View style={pdfStyles.piRow}>
+          <Text style={pdfStyles.piBorderCell}>
+            {data?.challenges}
+          </Text>
+        </View>
+      )}
+      <View style={pdfStyles.piRow}>
+        <Text style={[pdfStyles.piBorderCell, pdfStyles.fontWeightBoldAndLarge]}>
+          {strings.finalReportPlannedInterventionLessonsLearnt}
+        </Text>
+      </View>
+      {data?.lessons_learnt && (
+        <View style={pdfStyles.piRow}>
+          <Text style={pdfStyles.piBorderCell}>
+            {data?.lessons_learnt}
+          </Text>
+        </View>
+      )}
+    </View>
+  );
+}
+
+interface DrefOptions {
+  disaster_category: NumericKeyValuePair[];
+  needs_identified: StringKeyValuePair[];
+  planned_interventions: StringKeyValuePair[];
+  status: NumericKeyValuePair[];
+  type_of_onset: NumericKeyValuePair[];
+  users: {
+    id: number;
+    first_name: string;
+    last_name: string;
+    email: string;
+    username: string;
+  }[];
+}
+
+interface Props {
+  finalReportResponse: DrefFinalReportApiFields;
+  drefOptions: DrefOptions;
+  strings: Strings;
+}
+
+const logoUrl = '/assets/graphics/layout/go-logo-2020.png';
```

Comment:
Can we not import the png? The code may silently break when/if the path for logo changes.